ci/rt.sh

```diff
@@ -0,0 +1,254 @@
+#!/bin/bash
+######################################################################
+# This script is desined for UPP regression tests run by UPP developer.
+# Wen Meng, 12/2020, First version.
+# Fernando Andrade-Maldonado 5/2023 rework for CLI Options
+# Fernando Andrade-Maldonado / Wen Meng 9/2023 Add Hercules, fix typos, and refactor
+#
+######################################################################
+
+git_branch="develop"
+git_url="https://github.com/NOAA-EMC/UPP.git"
+clone_on="no"
+
+while getopts a:w:h:r:t:b:u:c opt; do
+  case $opt in
+    a) accnr=${OPTARG}
+        ;;
+    w) workdir=${OPTARG}
+        ;;
+    h) homedir=${OPTARG}
+        ;;
+    r) rundir=${OPTARG}
+        ;;
+    t) test_v=${OPTARG}
+        ;;
+    b) git_branch=${OPTARG}
+        ;;
+    u) git_url=${OPTARG}
+        ;;
+    c) clone_on="yes"
+	;;
+  esac
+done
+
+#UPP working copy
+test_v=${test_v:-`pwd`/../}
+if [[ $clone_on == "yes" ]]; then
+  rm -rf $test_v
+  mkdir -p $test_v
+  git clone -b $git_branch $git_url $test_v
+fi
+export svndir=${test_v}
+
+#Assume a nems account to run with
+accnr=${accnr:-"nems"}
+
+#Build UPP executable
+build_exe=yes
+
+#Choose run specific model
+run_nmmb=yes
+run_gfs=yes
+run_gefs=yes
+run_fv3r=yes
+run_rap=yes
+run_hrrr=yes
+run_hafs=yes
+run_rtma=yes
+
+#find machine
+mac=$(hostname | cut -c1-1)
+mac2=$(hostname | cut -c1-2)
+mac3=$(hostname | cut -c1-4)
+if [ $mac = v -o $mac = m  ] ; then   # WCOSS_D (venus and mars)
```

Comment:
@FernandoAndrade-NOAA Can you remove this if block for WCOSS_D since this platform was retired.

Reply:
Sure thing! With no explicit WCOSS2 check after removal of 64-67, I'm assuming you have an alternative test script on that machine you've been running RTs with?

Reply:
@FernandoAndrade-NOAA Good catching. You might add settings for WCOSS2 as:
```
elif [ $mac = d -o $mac = c ]; then    # WCOSS2
 export machine=WCOSS2
 export homedir=/u/wen.meng/noscrub/ncep_post/post_regression_test_new
 export rundir=/lfs/h2/emc/ptmp/$USER
module load prod_util
```

Reply:
Added, thanks!